### PR TITLE
ceph-build-arm64: Test for arm64 build, separate from ceph-build

### PR DESCRIPTION
**Pull request by dmick**
Some things want to be different in this build; let's keep them
away from the main ceph build until we're sure they can coexist
peacefully.

Signed-off-by: Dan Mick <dan.mick@redhat.com>

### CODE REVIEW

File: ceph-build-arm64/config/definitions/ceph-build.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+          values:
+            - xenial
+
+    builders:
+      - shell: |
+          echo "Cleaning up top-level workarea (shared among workspaces)"
+          rm -rf dist
+          rm -rf venv
+          rm -rf release
+      - copyartifact:
```
Comment (alfredodeza) [inline ceph-build-arm64/config/definitions/ceph-build.yml]: I understand this is mostly copied from `ceph-build` but we don't need to copy artifacts any more (woo hoo!)

**Comment by alfredodeza** [comment]
You have included the rpm scripts but this is only for Xenial. Is the plan to get arm64 for RPM servers to later on?

**Comment by dmick** [comment]
Yes, Centos is next; I just haven't tried to make that happen yet, as my build slaves are in worse shape for it.  (The build slaves are not installable yet except by physical disk construction.  :( )

File: ceph-build-arm64/config/definitions/ceph-build.yml (hunk 1 above)
Comment (dmick) [inline ceph-build-arm64/config/definitions/ceph-build.yml]: cool

**Pull request merged**